templates/common/infra/bicep/core/security/keyvault-secrets.bicep

```diff
@@ -0,0 +1,23 @@
+param tags object = {}
+param keyVaultName string
+param secrets array = []
+
+resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' existing = {
+  name: keyVaultName
+}
+
+@batchSize(1)
```

Comment:
Why are we setting batchsize to 1 here? Wouldn't we rather create them in parallel to speed up deployment?

Reply:
Removed the file